chore: release notes for 1.3.4 (#857)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/ref-release-notes-deprecated-functionalities.adoc

[id="deprecated-functionality-rhidp-1138"]
== `spec.application.image`, `spec.application.replicas` and `spec.application.imagePullSecrets` fields are deprecated

`spec.application.image`, `spec.application.replicas` and `spec.application.imagePullSecrets` fields are deprecated in `v1alpha2` in favour of `spec.deployment`. 

> Procedure:

Reviewer: Maybe format this as a heading?

To update your {product-short} Operation configuration:

> . Remove the `spec.application.image`, `spec.application.replicas` and `spec.application.imagePullSecrets` fields from the Operator configuration:

Reviewer: ```suggestion
. Remove the `spec.application.image`, `spec.application.replicas` and `spec.application.imagePullSecrets` fields from the Operator configuration. For example:

```

+
[source,yaml]
----
spec:
  application:
    replicas: 2 # &lt;1&gt;
    imagePullSecrets: # &lt;2&gt;
      - my-secret-name
    image: quay.io/my/my-rhdh:latest # &lt;3&gt;
----
&lt;1&gt; Replica count.
&lt;2&gt; Array of image pull secrets names.
&lt;3&gt; Image name.

> . Replace the removed fields by new `spec.deployment` fields, such as:

Reviewer: ```suggestion
. Replace the removed fields with new `spec.deployment` fields. For example:

```
